host-configs/LLNL/quartz-base.cmake

```diff
@@ -50,4 +50,9 @@ set(MKL_LIBRARIES ${MKL_ROOT}/lib/intel64/libmkl_intel_lp64.so
                   CACHE STRING "")
 
 
+# PYGEOSX
+set(ENABLE_PYGEOSX ON CACHE BOOL "")
+set(Python3_ROOT_DIR /usr/gapps/GEOSX/thirdPartyLibs/python/linux-rhel7-x86_64-gcc@8.1.0/python@3.8.5 CACHE PATH "")
+set(Python3_EXECUTABLE /usr/gapps/GEOSX/thirdPartyLibs/python/linux-rhel7-x86_64-gcc@8.1.0/python@3.8.5/bin/python3 CACHE PATH "")
+
```

Comment:
This needs to be updated for the current quartz install

Reply:
you actually already have this above in the file. I think we can just delete these lines.